src/styles/abstracts/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin letter-size-mobile($width, $height) {
+    height: $height;
+    width: $width;
+    font-size: 0.8 * $width;
+    line-height: 0.91 * $height;
+}
+
+@mixin letter-size-desktop($width, $height) {
```

Comment:
is it used anywhere? if not, then I'd say the code should be added when it's needed